### DISCUSSION TITLE
Key Blacklisting in EJBCA-ce

### DISCUSSION THREAD

**Discussion by hzami** [Keyfactor/ejbca-ce#632] [Q&A]
Hello EJBCA Community,

I am currently using EJBCA Community Edition installed via Docker, and I’m seeking some guidance on how to manage and understand key blacklisting within this environment.
My questions are:
 1/How Does Key Blacklisting Work in EJBCA CE?
        Can someone explain the underlying process of key blacklisting in EJBCA CE?
        What checks are performed during the certificate issuance process to enforce blacklisting?
  2/Verifying Blacklisted Keys:
        After importing a key blacklist, how can I verify that the keys are indeed blacklisted?
        Is there a way to list or view the blacklisted keys using the CLI or any other method?
 3/Best Practices and Recommendations:
        Are there any best practices or recommendations for managing key blacklists in EJBCA CE?
        How can I ensure that my blacklisting policies are effectively enforced?

**Comment by primetomas** [comment]
You mean the block list key validator right?
https://doc.primekey.com/ejbca/ejbca-operations/ejbca-ca-concept-guide/validators-overview/key-validators#KeyValidators-BlockListKeyValidator

**Comment by hzami** [comment]
yes

**Reply by primetomas** [reply]
Doesn’t the documentation answer at least 1 and 2? For 3 it shows how to use Debian weak key block lists. CAB Forum is on the way to create new block lists and the documentation will be updated for that after some time after that is ready. 
any more detailed questions?
